lib/modules/manager/gomod/artifacts.spec.ts

```diff
@@ -1291,4 +1297,190 @@ describe('modules/manager/gomod/artifacts', () => {
       },
     ]);
   });
+
+  it('GoConstraints when go.mod file content null', async () => {
```

Comment:
I don't understand this test name, can you try rephrasing it? I don't know what it's testing